packages/example-phone/src/scripts/components/call/dialer.js

```diff
@@ -31,7 +31,7 @@ export default class Dialer extends Component {
 
   render() {
     return (
-      <Form className="form-inline" onSubmit={this.handleSubmit.bind(this)} ref={(f) => {this.form = f;}}>
+      <Form formClassName="form-inline" onSubmit={this.handleSubmit.bind(this)} ref={(f) => {this.form = f;}}>
```

Comment:
No, there was an eslint rule added that components can't have `className` properties. In our implementation of the form, I pass it down to `<form className={formClassName}`